access/slack/config.go

```diff
@@ -1,56 +1,62 @@
 package main
 
 import (
+	"fmt"
 	"strings"
 
 	"github.com/gravitational/teleport-plugins/lib"
 	"github.com/gravitational/teleport-plugins/lib/logger"
+	"github.com/gravitational/teleport/api/types"
 	"github.com/gravitational/trace"
 	"github.com/pelletier/go-toml"
 )
 
 // Config stores the full configuration for the teleport-slack plugin to run.
 type Config struct {
-	Teleport lib.TeleportConfig `toml:"teleport"`
```

Comment:
providing the `toml:"..."` explicitly is only needed for non-lower-camel-case options. e.g. `role_to_recipients`.